radixdlt/build.gradle

```diff
@@ -85,7 +85,7 @@ jacocoTestReport {
 dependencies {
     compile 'io.reactivex.rxjava3:rxjava:3.0.0'
     compile 'com.sleepycat:je:18.3.12'
-    compile 'com.radixdlt:radix-engine-library:1.0-beta.7'
+    compile 'com.github.radixdlt:radix-engine-library:rc~1.0-beta.8-SNAPSHOT'
```

Comment:
Either com.radixdlt or com.github.radixdlt will work, as we have the right TXT records in place.
https://jitpack.io/docs/#custom-domain-name